packages/voicechat-server/tsconfig.json

```diff
@@ -0,0 +1,71 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
```

Comment:
Hi, please cleanup unused comments in this file and mark a comment that this subpackage doesn't have to join the project reference because it is totally isolate.